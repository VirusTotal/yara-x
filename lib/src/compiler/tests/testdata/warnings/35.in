rule test{
  strings:
    $a = /[A-Za-z]{2,}/
  condition:
    $a
}