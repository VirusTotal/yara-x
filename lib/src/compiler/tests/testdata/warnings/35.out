warning[slow_pattern]: slow pattern
 --> line:3:5
  |
3 |     $a = /[A-Za-z]{2,}/
  |     ------------------- this pattern may slow down the scan
  |
